masterlist.yaml

```
@@ -2915,6 +2915,12 @@ plugins:
       - 'fwv.esm'
       - 'fwv.esp'
       - 'FNC Overhaul Project.esp'
+    msg:
+      - <<: *compatNotes
+        subs: [ 'https://docs.google.com/document/d/14OpRnF5zW4x-qZMQ62JZgM5A4Xq02TAsDoKwKd41P_s/mobilebasic' ]
+      - <<: *compatIssuesWithX
+        subs: [ 'Fallout - The Frontier' ]
+        condition: file("NVSE/Plugins/frontier_tweaks.dll")
```

Just two changes required: messages should be sorted by alias alphabetically, & condition strings should be encapsulated in single quotes.
```suggestion
      - <<: *compatIssuesWithX
        subs: [ 'Fallout - The Frontier' ]
        condition: 'file("NVSE/Plugins/frontier_tweaks.dll")'
      - <<: *compatNotes
        subs: [ 'https://docs.google.com/document/d/14OpRnF5zW4x-qZMQ62JZgM5A4Xq02TAsDoKwKd41P_s/mobilebasic' ]
```